test/libsolidity/util/TestFunctionCall.cpp

```diff
@@ -208,6 +208,7 @@ string TestFunctionCall::formatBytesParameters(
 					" does not match the one inferred from ABI."
 				);
 			}
+	_errorReporter.warning("The call to \"" + _signature + "\" returned \n" + BytesUtils::formatRawBytes(_bytes));
```

Comment:
forgotten debug output? Or wrong indent?

Reply:
Ah, I believe this was added too early in the commit history

Reply:
What exactly do you mean? The indentation seems to be correct and it was added with the right commit.

Reply:
Your intention was to add this in commit https://github.com/ethereum/solidity/pull/7052/commits/e72e6964777f0768baa0c22fd476d56e9960db86
"[isoltest] Prints bytes result if expectations mismatch."? If yes, then it's not correctly indented for the
`for (auto const& param: preferredParams)` loop body.

Reply:
Ah, I see. You're right. It is supposed to be added by that commit. I would squash at least some commits, and this one for sure. So, shall I leave it like that?

Reply:
sgtm :+1: